api/app/routers/v1/projects.py

```diff
@@ -0,0 +1,33 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from typing import List
+from uuid import UUID
+from app.controllers.project import ProjectController
+from app.schemas.project_schemas import ProjectCreate, ProjectRead
+
+from app.routers.utilities import get_db_session
+
+router = APIRouter(prefix="/projects", tags=["projects"])
+
+
+@router.post("/", response_model=ProjectRead)
+def create_project(project_data: ProjectCreate, db: Session = Depends(get_db_session)):
```

Comment:
for all routes you'll need the current user dependency so you can 

1. check that the user has permission to execute the request
2. scope the result of the request

for example, `ProjectController` could take an `actor` param, and then `create_project` would create a project BY that user, belonging to that User's Org.